src/app/donation-start/donation-start-form/donation-start-form.component.html

```diff
@@ -302,19 +308,16 @@ <h3 class="b-rt-0 b-m-0 b-bold"><span class="span-hr"></span>How does Big Give u
             </div>
           }
           <div style="text-align: center">
-            <button
-              type="button"
-              class="continue"
-              mat-raised-button
-              color="primary"
-              (click)="progressFromStepGiftAid()"
-            >
-              Continue
-            </button>
+            <button type="button" class="continue" mat-raised-button color="primary" (click)="next()">Continue</button>
           </div>
         </mat-step>
       }
-      <mat-step formGroupName="payment" [stepControl]="paymentGroup" label="Payment details">
+      <mat-step
+        formGroupName="payment"
+        [stepControl]="paymentGroup"
+        label="Payment details"
+        (interacted)="continueFromPaymentStep()"
```

Comment:
won't this stop people going backwards from the payment step if they haven't filled in payment details? Presumably we should still allow backwards movement in that case just not forwards.

Reply:
hmm so tested, on this branch and it does allow me progress backwards, without filling in the payment step, but it also shows this error toast - we shouldn't be showing any error message if the user just wants to progress backwards.

<img width="1362" height="435" alt="image" src="https://github.com/user-attachments/assets/ad8e4300-754d-466c-ae97-fce9800cb043" />